src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
I'd also recommend removing `put` and renaming `putHelp` to `put`, since tuples in Elm aren't hard, and the compiler will give reasonable error messages in the case that a dev passes the wrong thing.